pkg/azurefile/azurefile.go

```diff
@@ -1031,3 +1121,63 @@ func (d *Driver) SetAzureCredentials(ctx context.Context, accountName, accountKe
 	}
 	return secretName, err
 }
+
+func (d *Driver) getAzcopyJob(dstFileshare string) (string, string, error) {
+	cmdStr := fmt.Sprintf("azcopy jobs list | grep %s -B 3", dstFileshare)
+	out, err := exec.Command("sh", "-c", cmdStr).CombinedOutput()
+	klog.V(2).Infof("get list jobs info: %v", string(out))
+	if err != nil && err.Error() != "exit status 1" {
+		return "", "", fmt.Errorf("couldn't list jobs in azcopy %v", err)
+	}
+	jobid, err := d.parseAzcopyJobList(string(out))
+	klog.V(2).Infof("parse azcopy jobid: %s", jobid)
+	if err != nil {
+		return "", "", fmt.Errorf("parse jobid in azcopy get error %v", err)
+	}
+	if jobid == "" {
+		return jobid, "", nil
+	}
+	percent, err := d.parseAzcopyJobSummary(jobid)
+	klog.V(2).Infof("parse azcopy summary, percent: %s", percent)
+	return jobid, percent, err
+}
+
+func (d *Driver) parseAzcopyJobList(out string) (string, error) {
```

Comment:
these functions could be in ultil function, and does not need to a member function of Driver.
paste the job string in the comments to show how this function works.

Reply:
move it to util.go